src/util/FactorUtil.js

```diff
@@ -186,7 +186,7 @@ define(['okta'], function (Okta) {
 
       var policyComplexity = policy.complexity;
       var requirements = _.map(policyComplexity, function (complexityValue, complexityType) {
-        if (!complexityValue) {
+        if (complexityValue <= 0) { // to skip 0 and -1
```

Comment:
In theory, the api won't return complexity as -1. However, this is a good check just in case.